routes/users.py

```diff
@@ -48,3 +48,17 @@ async def delete_user(identifier: models.UserIdentifier):
 async def get_user(identifier: models.UserIdentifier):
     user_data = await utils.get_user_info_by_identifier(identifier)
     return models.UserInfoQueryResponse(**user_data.dict())
+
+
+# this implementation doesn't use models.ValidateUserResponse. Should I implement this?
+# also, how can I define a 404 response if it user isn't found
```

Comment:
Good question. We can't actually define ahead of time what exceptions can be thrown, but we can just throw them as normal and they will be interpreted by the FastAPI code as `HTTPException`s and behave as expected in the frontend.

Take a look in the `models.exceptions` module for an example of an exception if you need a custom one. 
If one of the exceptions in that module fits your needs, then you can just `raise` it and it'll go all the way to the client and work as you expect. 

e.g.:
```python
if not user_document:
    raise exceptions.UserNotFoundException
```